tuist/tuist
# Add CrashReporter with targetSettings to fixtures

Resolves #3488
Request for comments document (if applies): -

### Short description 📝

As discussed in #3488, this PR adds the CrashReporter dependency with some `targetSettings` to the `app_with_spm_dependencies` fixtures.

### How to test the changes locally 🧐

1. Check out the project
2. Run `./fourier test tuist acceptance projects/tuist/features/dependencies.feature:11`

### Checklist ✅
